examples/dataset_example.py

```diff
@@ -9,13 +9,16 @@ def main():
     """
 
     review_dataset = ReviewDataset('Citalopram')
```

Comment:
Same comments as the classifier example. We should leave it on the Citalopram link. If we want to change the example to Adriamycin then we should change the name on line 11 too.